mobile/src/app/auth/verify.tsx

```
@@ -0,0 +1,100 @@
+          keyboardShouldPersistTaps="handled"
+          showsVerticalScrollIndicator={false}
+        >
+          <Text className="text-primary text-2xl font-semibold mb-4">
+            Verify Phone Number
+          </Text>
+          <Text className="text-primary text-lg mb-8">
+            Enter the verification code sent to {phone}
+          </Text>
+          <form.Field
+            name="code"
+            children={(field) => (
```

Biome (2.1.2)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)